components/tikv_util/src/memory.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use collections::{HashMap, HashSet};
+use kvproto::{
+    kvrpcpb::LockInfo,
+    metapb::{Region, RegionEpoch},
+    raft_cmdpb::ReadIndexRequest,
+};
+use std::collections::{BTreeMap, VecDeque};
+use std::hash::Hash;
+use std::mem;
+
+pub trait HeapSize {
```

Comment:
Can we directly use heapsize instead of implementing by ourselves

Reply:
Any suggestion?